thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -282,6 +282,29 @@ export function buildAnomalyStats(alertEvalMetrics, anomalyStats, showProjected
   return anomalyStats;
 }
 
+/**
```

Comment:
Per our conversation on slack, using local Storage is fine for launch. Eventually, we want to have more consistent code, and we'll move this into a service (a la rootcause)

Reply:
Awesome. Looking forward to that. thanks.